### PR TITLE
New session handling

### PR DESCRIPTION
**Pull request by marcelfolaron**
#### Link to ticket

Various
#2516
#2504

#### Description

Our custom session handler was outdated and needed to be adjusted. We are now using Laravel's ssession management

#### Checklist

- [ ] My code passes all test cases.
- [ ] My code passes our static analysis suite.
- [ ] My code passes our continuous integration process.

If your code does not pass the requirements on the checklist, you should add a comment explaining why this change 
should be exempt from the list.

### CODE REVIEW

File: app/Core/Application.php [hunk 1]
```diff
@@ -19,6 +33,46 @@ class Application extends Container
...
     private static bool $bootstrapped = false;
 
+    /**
+     * The deferred services and their providers.
+     *
+     * @var array
+     */
+    protected $deferredServices = [];
+
+    protected $serviceProviders = [];
```
Comment (pvi-github) [inline app/Core/Application.php:43]: Perhaps add an explaining comment for each ?

File: app/Core/Application.php [hunk 2]
```diff
@@ -19,6 +33,46 @@ class Application extends Container
...
+     * Constructor method for the class.
+     *
+     * Initializes the object and sets up the required dependencies and bindings.
+     * This method is automatically called when a new instance of the class is created.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+
```
Comment (pvi-github) [inline app/Core/Application.php:65]: Empty line ?

**Comment by pvi-github** [comment]
Has it been tested when php redis modules are not installed ?

**Comment by marcelfolaron** [comment]
> Has it been tested when php redis modules are not installed ?

Yes, redis is not required.

**Pull request merged**